scale-testing.mk

```diff
@@ -0,0 +1,63 @@
+# run the following from the project root:
+# make -f scale-testing.mk <make rule>
+
+# AWS variables for cloud hosted k8s testing
+AWS_REGION ?= us-east-2
+EKS_K8S_VERSION ?= 1.30
+
+# directories for cloud hosted k8s infrastructure for running tests
+# root directory for all integration tests
+TF_EKS_SRC_DIR ?= $(INTEGRATION_TEST_ROOT)/infra/scale-testing/eks-cluster
+TF_EKS_STATE_DIR ?= $(TF_EKS_SRC_DIR)/state
+TF_DEPLOY_SRC_DIR ?= $(INTEGRATION_TEST_ROOT)/infra/scale-testing/deployments
+TF_DEPLOY_STATE_DIR ?= $(TF_DEPLOY_SRC_DIR)/state
+
+include ./Makefile
+
+.PHONY: set-vault-license
+set-vault-license:
+ifeq ($(VAULT_ENTERPRISE), true)
+ifdef VAULT_LICENSE_CI
+	@echo Getting license from VAULT_LICENSE_CI
+	$(eval _VAULT_LICENSE=$(shell printenv VAULT_LICENSE_CI))
+else ifdef VAULT_LICENSE
+	@echo Getting license from VAULT_LICENSE
+	$(eval _VAULT_LICENSE=$(shell printenv VAULT_LICENSE))
+else ifdef VAULT_LICENSE_PATH
+	@echo Getting license from VAULT_LICENSE_PATH
+	$(eval _VAULT_LICENSE=$(shell cat $(VAULT_LICENSE_PATH)))
+	@test -n "$(_VAULT_LICENSE)" || ( echo vault license is empty; exit 1)
+else
+	$(error no valid vault license source provided, choices are VAULT_LICENSE_CI, VAULT_LICENSE, VAULT_LICENSE_PATH)
+endif
+endif
```

Comment:
Do we need another set-vault-license target here? Does the `include ./Makefile` not bring it in from the main makefile?

Reply:
We don't, you are right. Should be included, Iwill remove. Thank you